modin/core/storage_formats/pandas/query_compiler.py

```diff
@@ -410,23 +410,13 @@ def where(self, cond, other, **kwargs):
             cond, type(self)
         ), "Must have the same QueryCompiler subclass to perform this operation"
         if isinstance(other, type(self)):
-            # Note: Currently we are doing this with two maps across the entire
-            # data. This can be done with a single map, but it will take a
-            # modification in the `BlockPartition` class.
-            # If this were in one pass it would be ~2x faster.
-            # TODO (devin-petersohn) rewrite this to take one pass.
```

Comment:
It looks like we are no longer having to call binary_op twice. Are we seeing any performance improvements?

Reply:
I think so. I put some benchmark numbers in the PR description.

Reply:
@mvashishtha I don't see the numbers?